docs/tutorials/extracting_log_qgroundcontrol.rst

```diff
@@ -0,0 +1,115 @@
+
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _ifci_log:
+
+**************************************************************
+Exporting Logs from PX4 Flight Controller using QGroundControl
+**************************************************************
+
+.. note::
+    This tutorial is only applicable for PX4 firmware.
+
+
+Exporting Logs 
+==============
+
+Motor telemetry data can be found in the flight controller logs if you are using PX4 firmware.  
+Also, your PX4 firmware must have the EscStatus topic enabled for logging. Please refer to the `PX4 Logging documentation <https://docs.px4.io/main/en/dev_log/logging.html>`_ on how to configure which topics are logged.
+This tutorial covers how to extract and view motor telemetry logs from your flight controller's SD card using QGroundControl.
+
+.. note::
+    In this example, the PX4 firmware has IFCI enabled. Please refer to the documentation on :ref:`Setting up PX4 Firmware for Use with IFCI <ifci_px4_flight_controller>` to make sure your flight controller is configured properly if you are extracting IFCI data.
```

Comment:
I actually don't think you need to know what protocol is being used for any of the stuff on this page to make sense, I believe its all totally generic now, though maybe double check with Jordan. If that's true, you could probably just lose this note altogether, and just have the numbering section be the only time you mention specifics.